flake.nix

```diff
@@ -0,0 +1,62 @@
+{
+  description = "milestone34";
+  nixConfig = {
```

Comment:
I think it's a good idea to remove this `nixConfig` block altogether and put some instruction in the README instead. 

Quote from @brainrake:

> Don't do this, it's insecure.
> The only way to use trusted-public-keys from nix flake is if you put your user in trusted-users, which gives them > passwordless root.

Reply:
Yeah, exactly this. I got told the same thing yesterday.

Reply:
Ok. I thought living it with only `extra-substituters = [ "https://plutonomicon.cachix.org" ];` w/o `trusted-public-keys` is ok, as `https://plutonomicon.cachix.org` is public one.

Does anybody know how critical is `allow-import-from-derivation = "true";` here? (since this option was never enabled  in my `nix.conf`)

Reply:
Seems like `allow-import-from-derivation = "true";` is on by default: https://nixos.org/manual/nix/stable/command-ref/conf-file#conf-allow-import-from-derivation. So I removed whole `nixConfig` section.